spec/src/constructorio.js

```diff
@@ -29,6 +29,9 @@ describe(`ConstructorIO${bundledDescriptionSuffix}`, () => {
 
     if (bundled) {
       ConstructorIO = window.ConstructorioClient;
+
+      // Make global fetch available on the window
+      window.fetch = fetch;
```

Comment:
I think we might be able to move this to the `jsdom-global.js` utils file so we don't need to specify it in all these files.

Reply:
Fantastic idea! I forgot we have this file. Updating the code